RabbitMQ unable to write configuration file

Looks like there's a regression since version `3.7.26-management-alpine` was published. This affects our internal tests but is likely to affect users of the RabbitMQ container, hence posting this issue for visiblity:

```
org.testcontainers.containers.RabbitMQContainerTest > shouldMountConfigurationFile FAILED
    java.lang.AssertionError: 
    Expecting:
     <"touch: /etc/rabbitmq/rabbitmq-custom.conf: Permission denied
    WARNING: '/etc/rabbitmq/rabbitmq-custom.conf' is not writable, but environment variables have been provided which request that we write to it
      We have copied it to '/tmp/rabbitmq.conf' so it can be amended to work around the problem, but it is recommended that the read-only source file should be modified and the environment variables removed instead.
```

Downgrading the image used from `3.7-management-alpine` to `3.7.25-management-alpine` seems to work around the issue for now. I'll raise a PR to do this.

--- a/modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java
+++ b/modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java
@@ -28,7 +28,7 @@ public class RabbitMQContainer extends GenericContainer<RabbitMQContainer> {
      * The image defaults to the official RabbitmQ image: <a href="https://hub.docker.com/_/rabbitmq/">RabbitMQ</a>.
      */
     private static final String DEFAULT_IMAGE_NAME = "rabbitmq";
-    private static final String DEFAULT_TAG = "3.7-management-alpine";
+    private static final String DEFAULT_TAG = "3.7.25-management-alpine";
 
     private static final int DEFAULT_AMQP_PORT = 5672;
     private static final int DEFAULT_AMQPS_PORT = 5671;

--- a/modules/rabbitmq/src/test/java/org/testcontainers/containers/RabbitMQContainerTest.java
+++ b/modules/rabbitmq/src/test/java/org/testcontainers/containers/RabbitMQContainerTest.java
@@ -31,7 +31,7 @@ import org.testcontainers.utility.MountableFile;
  */
 public class RabbitMQContainerTest {
 
-    public static final String DEFAULT_IMAGE = "rabbitmq:3.7-management-alpine";
+    public static final String DEFAULT_IMAGE = "rabbitmq:3.7.25-management-alpine";
     public static final int DEFAULT_AMQPS_PORT = 5671;
     public static final int DEFAULT_AMQP_PORT = 5672;
     public static final int DEFAULT_HTTPS_PORT = 15671;

